src/styles.scss

```diff
@@ -9,3 +9,19 @@ body {
 ::first-letter {
   text-transform: capitalize;
 }
+
+.register__wrapper {
+  height: 70vh;
+}
+
+.login__wrapper {
+  height: 50vh;
+}
+
+.register__wrapper, .login__wrapper {
+  transition: all 0.5s ease-in-out;
+}
+
+.mat-dialog-container {
+  overflow: hidden !important;
```

Comment:
I think that overflow:hidden could work without !important if you added it to the class you appended to the dialog.

Reply:
Try to make it w/o !important.